encryptor/config/encryptionSettings.go

```diff
@@ -243,9 +254,15 @@ func (s *BasicColumnEncryptionSetting) Init() (err error) {
 	if s.ReEncryptToAcraBlock != nil && *s.ReEncryptToAcraBlock {
 		s.settingMask |= SettingReEncryptionFlag
 	}
+
+	tokenizeEnabled := (s.Tokenized != nil && *s.Tokenized)
+	if s.Tokenized != nil && !*s.Tokenized && s.TokenType != "" {
+		return errors.New("`tokenized` is disabled, but `token_type` is provided")
+	}
+
 	var tokenType common.TokenType
 	var ok bool
-	if s.TokenType != "" || s.Tokenized {
+	if s.TokenType != "" || tokenizeEnabled {
```

Comment:
why not just `if s.TokenType != ""`? Becase now only TokenType is required.

Reply:
Yeah, good point :thinking: